mappings/net/minecraft/client/render/ClientCauldronFluid.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_5636 net/minecraft/client/render/ClientCauldronFluid
```

Comment:
This class isn't related to cauldrons in any way.

Reply:
Solved in `f84bdc7`